packages/reference/src/resources/ExternalResourceCard/ExternalResourceCard.tsx

```diff
@@ -109,7 +167,12 @@ export function ExternalResourceCard({
             }
           : undefined
       }
-    />
+    >
+      <ExternalResourceCardDescription
+        subtitle={entity.fields.subtitle || `Product ID: ${entity.sys.id}`}
```

Comment:
`Product ID: ${entity.sys.id}` this must not belong here. We must not couple with ProductId label anywhere

Reply:
This was added because we don't send the subtitles from the Backend just yet and according to hande it is a necessary information to have.